src/graccreq/oim/OIMTopology.py

```diff
@@ -185,7 +211,8 @@ def check_probe(self, doc):
          dictionary if a match was not found
         """
         if 'ProbeName' in doc:
-            probe_fqdn = re.match('.+:(.+)', doc['ProbeName']).group(1)
+            probe_exp = re.compile('.+:(.+)')
```

Comment:
This should be compiled once, probably in the `__init__` function.  Then used below.  Having it hear compiles it each time this function is called.